pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -542,6 +547,22 @@ func appendStatusCauseForInvalidMasqueradeMacAddress(field *k8sfield.Path, cause
 	return causes
 }
 
+func appendStatusCauseForPasstWithoutPodNetwork(field *k8sfield.Path, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+	return append(causes, metav1.StatusCause{
+		Type:    metav1.CauseTypeFieldValueInvalid,
+		Message: "Passt interface only implemented with pod network",
+		Field:   field.Child("domain", "devices", "interfaces").Index(idx).Child("name").String(),
+	})
+}
+
+func appendStatusCauseForPasstWithMultipleInterfaces(field *k8sfield.Path, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+	return append(causes, metav1.StatusCause{
+		Type:    metav1.CauseTypeFieldValueInvalid,
+		Message: "Passt interface is only supported as the single interface of the VMI",
```

Comment:
what is the source of this limitation?

Reply:
Nothing interesting. Just started with it for the tech preview. Technically it should be possible to run a VM with multiple interfaces.
I found it a bit dangerous since all the interfaces will use that `qrap` `domain.Spec.Devices.Emulator`, that's why I preferred to start with this limitation.